### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by jonwinton** [tschaub/gulp-newer#77]
Is it possible to get a new release published to NPM?

Also happy to help maintain this package!

**Comment by Flimm** [comment]
Yes please. I would like to use the `ctime` option, which hasn't been released on `npm` yet.
